moto/core/common_models.py

```diff
@@ -94,8 +94,8 @@ def is_created(self) -> bool:
         return True
 
 
-class ConfigQueryModel:
-    def __init__(self, backends: Any):
+class ConfigQueryModel(Generic[SERVICE_BACKEND]):
+    def __init__(self, backends: BackendDict[SERVICE_BACKEND]):
```

Comment:
Nice - this is going to make development of future ConfigQueryModels much easier